src/main/java/io/github/wimdeblauwe/hsbt/mvc/HtmxMvcConfiguration.java

```diff
@@ -1,18 +1,35 @@
 package io.github.wimdeblauwe.hsbt.mvc;
 
+import java.util.List;
+
+import org.springframework.beans.factory.ObjectFactory;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnWebApplication;
 import org.springframework.boot.autoconfigure.web.servlet.WebMvcRegistrations;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.util.Assert;
 import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.servlet.LocaleResolver;
+import org.springframework.web.servlet.ViewResolver;
 import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 import org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerMapping;
 
-import java.util.List;
-
-@Configuration
+@Configuration(proxyBeanMethods = false)
 @ConditionalOnWebApplication
 public class HtmxMvcConfiguration implements WebMvcRegistrations, WebMvcConfigurer {
+
+    private final ViewResolver resolver;
+    private final ObjectFactory<LocaleResolver> locales;
+
+    HtmxMvcConfiguration(ViewResolver resolver, ObjectFactory<LocaleResolver> locales) {
```

Comment:
@odrotbohm Test are failing with`No qualifying bean of type 'org.springframework.web.servlet.ViewResolver' available: expected single matching bean but found 5: beanNameViewResolver,mvcViewResolver,defaultViewResolver,viewResolver,thymeleafViewResolver`

Any recommendations for selecting the view resolved?

Reply:
Good one, and sorry for not actually trying this here locally. I've just pushed an update that refers to `ThymeleafViewResolver` explicitly, which should be fine (and was what I was using in my original prototype) as we assume Thymeleaf to be present anyway.